appengine/standard/firebase/firetactoe/templates/_firebase_config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
You're absolutely right. I missed this in dev b/c I had both files in there. I'm tempted to just use your technique given simpler setup.